hms-app/src/components/card-details/IdeaDetails.tsx

```diff
@@ -253,6 +253,14 @@ export default function IdeaDetails(props: IProps) {
     }
   }
 
+  const ideaCreationDate = () => {
+    return (
+      <Text className={classes.smallText}>
+        {new Date(idea.creationDate).toDateString()}
```

Comment:
can we add an "Creation Date: " before the actual date so that is 100% clear to the user what this date is